draft-ietf-quic-transport.md

```diff
@@ -3685,14 +3691,18 @@ packet with the largest packet number is received and the time an acknowledgment
 is sent.  The endpoint encodes this acknowledgement delay in the ACK Delay field
 of an ACK frame; see {{frame-ack}}.  This allows the receiver of the ACK frame
 to adjust for any intentional delays, which is important for getting a better
-estimate of the path RTT when acknowledgments are delayed.  A packet might be
-held in the OS kernel or elsewhere on the host before being processed.  An
-endpoint MUST NOT include delays that it does not control when populating the
-ACK Delay field in an ACK frame.
-
-Since the acknowledgement delay is not used for Initial and Handshake
-packets, the ACK Delay field in acknowledgements for those packet types
-SHOULD be set to 0.
+estimate of the path RTT when acknowledgments are delayed.
+
+A packet might be held in the OS kernel or elsewhere on the host before being
```

Comment:
I'd still like to strike the word 'intentionally' from above, since I don't think it's helpful.

Reply:
I'm leaving it there. I don't want to make any more editorial changes to existing text in this PR.

Reply:
Aside from "intentionally" being existing text, I would argue that we have to keep that word. As we have discussed elsewhere (e.g., #3970), any delay that is *not* intentionally introduced has to be accounted as path delay. As an example, [a delay caused by the timer waking up late](https://github.com/quicwg/base-drafts/pull/3874/#discussion_r466428681) should ideally be considered as part of the path delay (and hence be *not* included in the reported ack delay).